### ISSUE TITLE
Allow Topic Author to Edit Title

### ISSUE THREAD

**Issue by TheYorkshireDev** [POPWorldMedia/POPForums#80]
If a user who is not a moderator creates a topic on the forum which has a typo in the title, there is no way that the author can update the title afterwards.

I propose, we allow the author to edit the topic title by editing the title on their **first** post. Currently, each post has a title which to my knowledge is not used anywhere other on post creation taking the form `Re: Topic Title` for replies.

**Comment by jeffputz** [comment]
You are correct, it's not used elsewhere. Right now, topic title edits are made in the context of the moderator box. Do you think it goes with the edit for the initial post?

**Comment by TheYorkshireDev** [comment]
I implemented this enhancement after a user requested it in my fork of the Forum for work. They actually suggested editing the original post and updating the title, which is what you do when updating a title on StackOverflow.

One thing I did for this was to show keep the title text field (which you hid on a145954bba281bb18f0982aa40bbe46201d384be) on edit and reply for users to see the "Re: PostTitle" but have it disabled.
